tests/integ/scala/test_dataframe_reader_suite.py

```diff
@@ -594,12 +594,10 @@ def test_read_csv_with_format_type_options(session, mode, local_testing_mode):
     assert res == [Row(1, "one", 1.2), Row(2, "two", 2.2)]
 
     # test when user does not input a right option:
-    df2 = get_reader(session, mode).schema(user_schema).csv(test_file_csv_colon)
-    with pytest.raises(SnowparkSQLException) as ex_info:
-        df2.collect()
-    assert (
-        "SQL compilation error" if not local_testing_mode else "Invalid stage"
-    ) in str(ex_info)
+    with pytest.raises(
+        ValueError, match="DataFrameReader can only read files from stage locations."
+    ):
+        get_reader(session, mode).schema(user_schema).csv(test_file_csv_colon)
```

Comment:
do you think this is a error BCR that previously customer gets SnowparkSQLException but now it's ValueError?